### PR TITLE
simplify quickstart

### PR DESCRIPTION
**Pull request by tobiaslins**
## Pull request checklist

- [x] All tests pass. Demo project builds and runs.

## What's in this pull request?

> It should be able to build the project by typing `npm run build` and `yarn build` instead of writing the `node_modules/.bin/gulp` path

### CODE REVIEW

**Comment by paberr** [comment]
Looks good to me. Thank you a lot for this contribution!
I will test that locally and if it works fine, we can merge it.

File: README.md [hunk 1]
```diff
@@ -73,21 +71,18 @@ Developers are free to choose between npm and yarn for managing the dependencies
...
 #### Run Testsuite
-- `gulp test` runs the testsuite in your browser.
-- `jasmine` runs the testsuite in NodeJs.
+- `yarn test` or `npm test` runs browser and NodeJS tests.
+- `yarn test-browser` or `npm run test-browser` runs the testsuite in your browser only.
+- `yarn test-node` or `npm run test-node` runs the testsuite in NodeJS only.
 
 #### Run ESLint
-`gulp eslint` runs the ESLint javascript linter.
+`yarn lint` runs the ESLint javascript linter.
```
Comment (mar-v-in) [inline README.md]: Please also mention npm here, (unfortunately) not everyone is on yarn yet

File: README.md [hunk 2]
```diff
@@ -73,21 +71,18 @@ Developers are free to choose between npm and yarn for managing the dependencies
...
 
 #### Build
-Executing `gulp build` concatenates all sources into `dist/{web,web-babel,web-crypto,node}.js`
+Executing `yarn build` concatenates all sources into `dist/{web,web-babel,web-crypto,node}.js`
```
Comment (mar-v-in) [inline README.md]: and here

File: README.md (hunk 1 above)
Comment (tobiaslins) [inline README.md]: done :)

File: README.md (hunk 2 above)
Comment (tobiaslins) [inline README.md]: done, thanks!

File: README.md [hunk 3]
```diff
@@ -79,10 +79,10 @@ Developers are free to choose between npm and yarn for managing the dependencies
 - `yarn test-node` or `npm run test-node` runs the testsuite in NodeJS only.
 
 #### Run ESLint
-`yarn lint` runs the ESLint javascript linter.
+`yarn lint` or `npm run lint` runs the ESLint javascript linter.
```
Comment (paberr) [inline README.md]: For conistency, I would now list them in reverse order, i.e., npm first, yarn second. This is also the order in which we list them before.

Comment (tobiaslins) [inline README.md]: makes sence - done

**Comment by paberr** [comment]
I finally had time to test this. But apparently `npm build` does not work. Perhaps `build` is a built-in keyword you cannot use there?

**Comment by louisjordan** [comment]
I think you're looking for `npm run build`

File: README.md [hunk 4]
```diff
@@ -78,21 +76,18 @@ Developers are free to choose between npm and yarn for managing the dependencies
...
+- `npm run test-browser` or `yarn test-browser` runs the testsuite in your browser only.
+- `npm run test-node` or `yarn test-node` runs the testsuite in NodeJS only.
 
 #### Run ESLint
-`gulp eslint` runs the ESLint javascript linter.
+`npm run lint` or `yarn lint` runs the ESLint javascript linter.
 
 #### Build
-Executing `gulp build` concatenates all sources into `dist/{web,web-babel,web-crypto,node}.js`
+Executing `npm build` or `yarn build` concatenates all sources into `dist/{web,web-babel,web-crypto,node}.js`
```
Comment (paberr) [inline README.md]: This should probably be `npm run build`.

File: README.md [hunk 5]
```diff
@@ -78,21 +76,18 @@ Developers are free to choose between npm and yarn for managing the dependencies
...
-If you decided on using yarn for managing the dependencies,
-you have to use `node_modules/.bin/gulp` instead of `gulp`
-and `node_modules/.bin/jasmine` instead of `jasmine` in the following.
-
 ### Test and Build
 
 #### Run Testsuite
-- `gulp test` runs the testsuite in your browser.
-- `jasmine` runs the testsuite in NodeJs.
+- `npm test` or `yarn test` runs browser and NodeJS tests.
```
Comment (paberr) [inline README.md]: `npm test` seems to work. No change required here.

**Comment by paberr** [comment]
Thank you @louisjordan ! That indeed fixes the problem.

**Comment by tobiaslins** [comment]
Sorry - forgot that this `npm build` is not working (its reserved)! Should be fixed now with `npm run build`

**Pull request merged**